test/quantization/test_quantize_fx.py

```diff
@@ -457,7 +457,7 @@ def forward(self, x):
 
         tests = [
             (Linear, (linear_weight,), (linear_input,),
-             ns.call_function(torch.ops.quantized.linear_dynamic),
+             ns.call_function(torch.ops.quantized.linear_dynamic_fp16),
```

Comment:
Do we need to update any other tests? I see few others with float16_dynamic_qconfig

Reply:
checked the tests, looks like we only need to update this one